.github/workflows/build_and_test.yml

```diff
@@ -365,9 +365,11 @@ jobs:
           - >-
             pyspark-connect
           - >-
-            pyspark-pandas-connect
+            pyspark-pandas-connect-part0
           - >-
-            pyspark-pandas-slow-connect
+            pyspark-pandas-connect-part1
```

Comment:
@itholic can you rebalance this? This test alone takes 3 hours while others all finish within 2 hours.